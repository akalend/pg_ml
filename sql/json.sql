select ml_json_info('astra.json');
select * from ml_json_info('boston.json');
select * from ml_json_info('titanic.json');