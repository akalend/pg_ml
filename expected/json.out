select ml_json_info('astra.json');
   ml_json_info   
------------------
 (g,float)
 (delta,float)
 (r,float)
 (plate,float)
 (redshift,float)
 (u,float)
 (MJD,float)
 (z,float)
 (fiber_ID,float)
 (alpha,float)
 (cam_col,float)
 (i,float)
(12 rows)

select * from ml_json_info('boston.json');
  feature   | type  
------------+-------
 zn         | float
 indus      | float
 chas       | float
 black      | float
 lstat      | float
 Unnamed: 0 | float
 rad        | float
 tax        | float
 nox        | float
 rm         | float
 dis        | float
 age        | float
 ptratio    | float
 crim       | float
(14 rows)

select * from ml_json_info('titanic.json');
   feature   | type  
-------------+-------
 Pclass      | text
 PassengerId | text
 Cabin       | text
 Embarked    | text
 Ticket      | text
 Age         | float
 Name        | text
 Parch       | text
 SibSp       | text
 Sex         | text
 Fare        | float
(11 rows)

